apps/utils/prompt.py

```diff
@@ -21,27 +22,47 @@ class PromptVars(models.TextChoices):
     AgentTools.UPDATE_PARTICIPANT_DATA: [PromptVars.PARTICIPANT_DATA],
 }
 
+PROMPT_VAR_CONTEXT_VAR_MAP = {
```

Comment:
This could use a comment to indicate its purpose

Reply:
Removed the need for this [here](https://github.com/dimagi/open-chat-studio/pull/1323/commits/f54b61dffe3f20c0e34c28706fd900fe62981f01)